models/project-extension/README.md

```diff
@@ -0,0 +1,26 @@
+# @commercetools-test-data/project-extension
+
+This package provides the data model for the commercetools internal API `ProjectExtension` type
+
+# Install
+
+```bash
+$ pnpm add -D @commercetools-test-data/project-extension
+```
+
+# Usage
+
+The package provides models for ProjectExtension.
+
+```ts
+import {
+  ProjectExtension,
+  type TProjectExtension,
+  TProjectExtensionGraphql,
+} from '@commercetools-test-data/project-extension';
```

Comment:
nit: only for consistency
```suggestion
import {
  ProjectExtension,
  type TProjectExtension,
  type TProjectExtensionGraphql,
} from '@commercetools-test-data/project-extension';
```

Reply:
updated [0540eed](https://github.com/commercetools/test-data/pull/562/commits/0540eed56dc1eede9d0529dce34832771d5bd58b)